plyer/platforms/linux/bluetooth.py

```diff
@@ -0,0 +1,22 @@
+from subprocess import Popen, PIPE
+from plyer.facades import Bluetooth
+from plyer.utils import whereis_exe
+import subprocess
+from os import environ
+
+
+class LinuxBluetooth(Bluetooth):
+
+    def _start(self):
+        subprocess.Popen(["service", "bluetooth", "start"])
```

Comment:
An app should not be starting/stopping a system service. Doing this also requires root access. The specific command here is based on the Upstart init system, which is used in older Debian/Ubuntu systems and has since been replaced by systemd. While the Upstart commands still work for backwards compatibility, I wouldn't count on that always being the case. (In fact, systemd is the only reason this command works now. Upstart's `service' would not have asked for a password, and MUST be run via sudo/gksudo instead.)

This also doesn't actually turn Bluetooth on/off, it just pairs/unpairs with preconfigured devices. `rfkill block`/`unblock` should be used to actually turn BT on/off. It should not require root access if the user has appropriate permissions.